security/sec-iride/src/test/java/org/geoserver/security/iride/identity/IrideIdentityTokenizerTest.java

```diff
@@ -80,7 +79,7 @@ public void testTokenizationThrowNullPointerExceptionForNullValue() throws Iride
      */
     @Test(expected = IrideIdentityMissingTokenException.class)
     public void testTokenizationFailForEmptyOrBlankValue() throws IrideIdentityMissingTokenException {
-        final String value = StringUtils.repeat(" ", RandomUtils.nextInt(10));
+        final String value = randomBlankOrEmptyString();
```

Comment:
Using random inputs inside a test is not a good practice. Tests should be predictable.